php/ClassInfo/ComposerClassmapFileRepository.php

```diff
@@ -104,13 +100,18 @@ public function find($path_pattern, ClassFilter $filter = null)
      */
     private function isValid($classpath)
     {
-        $filePath = $this->classLoader->findFile($classpath);
```

Comment:
What phpcd.vim should depend on is  not the composer by the autoload mechanism it offered. If a php project has a autoload mechanism, even if it is not according to the PSR, phpcd.vim will happy to support it, theoretically. However, If we introduce the new dependence to the composer's classloader, phpcd.vim will become to composer only.